src/App.js

```diff
@@ -31,21 +33,24 @@ function App() {
     fetchData();
   }, []);
   return (
-    <StyledApp>
-      {isLoading ? (
-        <p>Loading...</p>
-      ) : (
-          <Switch>
-            <Route exact path="/">
-              <ConferenceList conferences={conferences} />
-            </Route>
-            <Route path="/:id">
-              <ConferenceDetails conferences={conferences} />
-            </Route>
-          </Switch>
-        )}
-      {errorCaught ? <p>An error has occurred...</p> : ""}
-    </StyledApp>
+    <BrowserRouter>
+      <StyledApp>
+        <Navbar />
+        {isLoading ? (
+          <p>Loading...</p>
+        ) : (
+            <Switch>
+              <Route exact path="/">
+                <ConferenceList conferences={conferences} />
+              </Route>
+              <Route path="/:id">
+                <ConferenceDetails conferences={conferences} />
+              </Route>
+            </Switch>
+          )}
+        {errorCaught ? <p>An error has occurred...</p> : ""}
```

Comment:
Another way (maybe a little bit more elegant) to write this is { errorCaught && <p>An error has occurred...</p> } so as long as there's no error nothing will be rendered and as soon is there's one it will render the <p>.

Doesn't have to be changed as part of the PR or at all though ;)

Reply:
ok thank you!! :)